lib/chat_web/live/helpers/uploader.ex

```diff
@@ -179,7 +179,7 @@ defmodule ChatWeb.LiveHelpers.Uploader do
       entry.client_last_modified
     ]
     |> Enum.join(":")
-    |> Enigma.hash()
+    |> Enigma.short_hash()
```

Comment:
Should be just `Enigma.hash`

 `Base.encode16(case: :lower)` to get string
But use it on the edge of system. I.e when you need url or JSON